modules/cli/src/main/scala/scala/cli/commands/publish/Publish.scala

```diff
@@ -56,13 +56,7 @@ import scala.cli.commands.shared.{
 import scala.cli.commands.util.{BuildCommandHelpers, ScalaCliSttpBackend}
 import scala.cli.commands.{ScalaCommand, SpecificationLevel, WatchUtil}
 import scala.cli.config.{ConfigDb, Keys, PasswordOption, PublishCredentials}
-import scala.cli.errors.{
-  FailedToSignFileError,
-  MalformedChecksumsError,
-  MissingConfigEntryError,
-  MissingPublishOptionError,
-  UploadError
-}
+import scala.cli.errors._
```

Comment:
NIT might be overkill to import all errors here. Also, use `*` syntax, if you leave it, `_` is to be deprecated soon